### ISSUE TITLE
TA Feedback - Alex Miller

### ISSUE THREAD

**Issue by alexjmiller5** [KaiSolter/cs506_final_project#1]
Hello,

I think this idea is cool, is will be able to see how good ELO can predict somebody's chess ability compared to considering all their chess characteristics.

I think that gradient boosting and neural networks are both well suited for classification tasks such as the one you have in mind. Neural networks might be a little bit more complicated to setup compared to gradient descent but will also probably perform better than gradient descent.
